src/site/content/en/blog/form-fatigue-reduce-number-of-fields/index.md

```diff
@@ -0,0 +1,400 @@
+---
+title: "Reduce form fatigue: reduce the number of fields and steps"
+author: mdiblasio
+date: 2019-05-03
+description: |
+  This post describes how to reduce the number of field and steps in forms.
+wf_blink_components: Blink>Accessibility
+tags:
+  - post
+  - accessibility
+  - ux
+---
+
+<!-- TODO (): update tags -->
+
+_This is post 3 of 5 of the [Reduce form fatigue series](../form-fatigue) that
+shows you how to build better forms for online stores._
+
+<!-- TODO (robdodson): is there a format we should use for this line that opens each article?  -->
+
+<!-- TODO (dutton): is there a format we should use for this line that opens each article?  -->
+
+The best forms are the ones with the fewest inputs. 
+
+
+<div class="w-stats">
+  <div class="w-stat">
+    <p class="w-stat__figure">26<sub class="w-stat__sub">%</sub></p>
+    <p class="w-stat__desc"><a href="https://baymard.com/lists/cart-abandonment-rate">Of users abandon orders
+due to a "too long / complicated checkout process"</a></p>
+  </div>
+  <div class="w-stat">
+    <p class="w-stat__figure">13<sub class="w-stat__sub">%</sub></p>
+    <p class="w-stat__desc"><a href="https://baymard.com/lists/cart-abandonment-rate">Of users abandon a site or
+app because there are too many steps to purchase</a></p>
+  </div>
+  <div class="w-stat">
+    <p class="w-stat__figure">6-8</p>
+    <p class="w-stat__desc"><a href="https://baymard.com/blog/checkout-flow-average-form-fields">Average number of form fields on top performance e-commerce sites</a></p>
+  </div>
+</div>
+
++   **Consolidate fields where possible**. For example, ask users for
+    their full name instead of first and last names separately:
+
+<figure class="w-figure">
```

Comment:
I can definitely see how the side-by-side comparison would be helpful for a post like this.

That point aside, I found I had a tough time reading through these images because they're often comparing different sites. I realize it's a lot of extra work, but I think the examples would be much more effective if they were all built using the same UI elements. Then the before/after is much more immediately obvious to the reader.